CHANGELOG.asciidoc

```diff
@@ -87,6 +87,7 @@ https://github.com/elastic/beats/compare/v1.2.2...v1.2.3[View commits]
 *Topbeat*
 
 - Fix high CPU usage when using filtering under Windows. {pull}1598[1598]
+- Fix high values of proc.cpu.user_p. {pull}1928[1928]
```

Comment:
@monicasarbu I think this changelog entry was placed/merged to the wrong location, right? Shouldn't it be under the 1.3.0 section?